packages/create-redwood-app/template/api/src/lib/db.ts

```diff
@@ -0,0 +1,78 @@
+// See https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/constructor
```

Comment:
Update to `main` for a new version :)

Reply:
Is this in main currently:

```js
// See https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/constructor
// for options.

import { PrismaClient } from '@prisma/client'

export const db = new PrismaClient()
```

Oh, you have changed it to be `db.ts` now, too.  I see :+1:

Ok, so I think this will merge fine, right @peterp ?

Reply:
Perhaps no: 

Need to resolve conflicting files on merge:


> packages/create-redwood-app/template/api/src/lib/db.ts

Reply:
I merged this a few days ago, sorry dude.